frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -174,14 +169,6 @@ export default class Select extends Component {
     return <span style={{ minWidth: MIN_ICON_WIDTH }} />;
   };
 
-  handleClose = () => {
-    // Focusing in the next tick prevents it is from reopening
-    // when closed by selecting an item with Enter
-    setTimeout(() => {
-      this.selectButtonRef.current?.focus();
-    }, 0);
-  };
-
```

Comment:
I tested that selecting an item with an <kbd>Enter</kbd> won't reopen the popover, so I removed the function.